include/silo/common/fasta_reader.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SILO_FASTA_READER_H
+#define SILO_FASTA_READER_H
+
+#include "iostream"
+
+#include "silo/common/input_stream_wrapper.h"
+
+namespace silo {
+class FastaReader {
+  private:
+   silo::InputStreamWrapper in_file;
+
+  public:
+   explicit FastaReader(std::string in_file_name);
+
+   bool nextKey(std::string& key);
+
+   bool next(std::string& key, std::string& genome);
+
+   void reset();
```

Comment:
Do we need that method?

Reply:
Yes, the sorting method needs to read the file twice. I found this better than reopening the file a second time